queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -62,19 +63,62 @@ def update_affiliation(business: Business, filing: Filing):
         rv = AccountService.create_affiliation(
             account=bootstrap.account,
             business_registration=business.identifier,
-            business_name=business.legal_name
+            business_name=business.legal_name,
+            corp_type_code=business.legal_type
         )
 
         if rv not in (HTTPStatus.OK, HTTPStatus.CREATED):
             deaffiliation = AccountService.delete_affiliation(bootstrap.account, business.identifier)
+        else:
+            # flip the registration
+            # recreate the bootstrap, but point to the new business in the name
+            old_bs_affiliation = AccountService.delete_affiliation(bootstrap.account, bootstrap.identifier)
+            new_bs_affiliation = AccountService.create_affiliation(
+                account=bootstrap.account,
+                business_registration=bootstrap.identifier,
+                business_name=business.identifier,
```

Comment:
just verifying - this var is supposed to be business.identifier and not business.legal_name?

Reply:
This is the bootstrap getting deleted/added (as updates aren't. allowed) with the new business identifier, so that the dashboard can filter it out.